etc/adminhtml/system.xml

```diff
@@ -48,6 +48,16 @@
                     <frontend_model>Extend\Integration\Model\Config\Frontend\EnableProductProtection</frontend_model>
                     <backend_model>Extend\Integration\Model\Config\Backend\EnableProductProtection</backend_model>
                 </field>
+                <field id="enable_cart_offer" translate="label" type="select" sortOrder="20" showInDefault="1" canRestore="1">
+                    <label>Display Product Protection Offers in Cart</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <depends>
+                        <field id="enable">1</field>
+                    </depends>
```

Comment:
the `<depends/>` element means that this field will only be visible if the neighboring `enable` toggle is true - i.e., if the user has elected to enable Product Protection. if Product Protection is disabled, this field will disappear, and will be saved as `No`/`0` when config is saved.

Reply:
yeah, i think it should be ok - i believe `depends` can only access sibling fields within a group, unless an absolute `<section>/<group>/<field>` path is passed to it